Fix build failure on x86_64

When the target_cpu is x86_64, we should assume that the pkg uses 64bit,
only if USE_N32 is set, we can assume that the pkg uses 32bit. It used a
opposite logic before.

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

Upstream-Status: Pending
---
 configure.in |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

Index: nspr/configure.in
===================================================================
--- nspr.orig/configure.in
+++ nspr/configure.in
@@ -1875,28 +1875,24 @@ tools are selected during the Xcode/Deve
         PR_MD_ASFILES=os_Linux_ia64.s
         ;;
     x86_64)
-        if test -n "$USE_64"; then
-            PR_MD_ASFILES=os_Linux_x86_64.s
-        elif test -n "$USE_X32"; then
+        if test -n "$USE_X32"; then
+            AC_DEFINE(i386)
             PR_MD_ASFILES=os_Linux_x86_64.s
             CC="$CC -mx32"
             CXX="$CXX -mx32"
         else
-            AC_DEFINE(i386)
-            PR_MD_ASFILES=os_Linux_x86.s
-            CC="$CC -m32"
-            CXX="$CXX -m32"
+            PR_MD_ASFILES=os_Linux_x86_64.s
         fi
         ;;
     ppc|powerpc)
         PR_MD_ASFILES=os_Linux_ppc.s
         ;;
     powerpc64)
-        if test -n "$USE_64"; then
+        if test -n "$USE_N32"; then
+            PR_MD_ASFILES=os_Linux_ppc.s
+        else
             CC="$CC -m64"
             CXX="$CXX -m64"
-        else
-            PR_MD_ASFILES=os_Linux_ppc.s
         fi
         ;;
     esac    
